apps/dex/overlays/nishir-tailnet/kustomization.yaml

```
@@ -14,3 +16,16 @@ secretGenerator:
   - name: dex-config
     files:
       - config.yaml=dex/config.enc.yaml
+patches:
+  - path: patch-netpol.yaml
+    target:
+      name: dex
+      version: v1
+      kind: NetworkPolicy
+patches:
```

YAMLlint (1.37.1)

[error] duplication of key "patches" in mapping

(key-duplicates)